crates/subspace-fraud-proof/src/lib.rs

```diff
@@ -239,14 +241,34 @@ where
             ..
         } = proof;
 
+        let domain_id = *domain_id;
         let at = PBlock::Hash::decode(&mut parent_hash.encode().as_slice())
             .expect("Block Hash must be H256; qed");
-        let wasm_bundle = self
+        let system_wasm_bundle = self
             .client
             .runtime_api()
             .system_domain_wasm_bundle(&BlockId::Hash(at))
             .map_err(VerificationError::RuntimeApi)?;
 
+        let wasm_bundle = match domain_id {
```

Comment:
Given it's used only once, probably simler this way so that one line can be saved.

```suggestion
        let wasm_bundle = match *domain_id {
```